src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -296,6 +305,22 @@ def __init__(
         self.mane_transcript_mappings = mane_transcript_mappings
         self.liftover = liftover
 
+    def _get_mane_status_transcript(self, tx_ac: str) -> TranscriptPriority:
```

Comment:
I think this helper method might be best in the mane transcript mappings class

Reply:
We might want to rename to `_get_transcript_status` since longest compatible remaining is not technically a mane status.